lib/std/fs/file.zig

```diff
@@ -29,6 +29,15 @@ pub const File = struct {
 
     pub const Mode = os.mode_t;
 
+    /// The type that is used to represent the inode/file index number. On windows this is a
+    /// LARGE_INTEGER (i64), and on linux this is a u64.
+    pub const INode = switch (builtin.os.tag) {
+        .windows => os.windows.LARGE_INTEGER,
+        // TODO: Handle possibility of 128 bit numbers? ReFS on windows server 2012 uses a 128 bit file
+        // index. See https://docs.microsoft.com/en-us/windows/win32/api/fileapi/ns-fileapi-by_handle_file_information
+        else => u64,
```

Comment:
We actually do have a `std.os.ino_t` type, although the definition may be missing for some posix targets.

Reply:
I think in this case, even windows can define ino_t in lib/std/os/bits/windows.zig (similar to how fd_t is defined as HANDLE), and then we don't need a switch here, probably not even `INode`. The Stat struct could just use std.os.ino_t.